### DISCUSSION TITLE
How to request a cross domain resource in web browser?

### DISCUSSION THREAD

**Discussion by ismlsmile** [AvaloniaUI/Avalonia#14733] [General]
I want to show a web image in application, and I use httpclient. The same code is ok on windows and android, but it failed on web browser, I guess it is a cross domain problem. How to do this ?

**Comment by thevortexcloud** [comment]
It's a restriction of your web browser. Standard HTTP CORS applies. Microsoft has a somewhat brief overview of various web assembly limitations here:

https://github.com/dotnet/runtime/blob/main/src/mono/wasm/features.md#fetch---http-client

You probably need to change the server settings. But I am not sure how as I have not touched wasm that much.
